stable/prometheus-operator/values.yaml

```diff
@@ -561,6 +561,9 @@ prometheusOperator:
     labels: {}
     clusterIP: ""
 
+    ## To be used with a proxy extraContainer port
+    targetPort: 9090
```

Comment:
Why change this from the name to the number?

Reply:
To be more readable which port 
Case: anyone can use `web` name into extra container

Reply:
In this case, the same information needs to be added to the values file in the ci folder please

Reply:
done, added also to ci test value file